Karatsuba_Algorithm/Karatsuba_Algorithm.php

```diff
@@ -0,0 +1,50 @@
+<?php

```

Comment:
please remove this file

Reply:
But that's my previous pull request for issue #2248

Reply:
this file should be in another PR.

Reply:
@raghav-dalmia  already deleted the Karatsuba_Algorithm.php file . Tell me if I have to do something else